src/main/java/lotto/domain/entity/LottoTickets.java

```diff
@@ -0,0 +1,32 @@
+package lotto.domain.entity;
+
+import java.util.ArrayList;
+import java.util.List;
+import lotto.config.LottoRank;
+import lotto.util.RandomNumbers;
+
+public class LottoTickets {
+    private List<Lotto> tickets = new ArrayList<>();
+
+    public void addTicket(Lotto ticket) {
+        tickets.add(ticket);
+    }
+
+    public LottoResults aggregateLottoResults(WinningNumbers winningNumbers) {
+        LottoResults lottoResults = new LottoResults();
+
+        for (Lotto lotto : this.tickets) {
+            int matchCount = lotto.countMatch(winningNumbers);
+            boolean matchBonus = lotto.containsBonusNumber(winningNumbers);
+
+            LottoRank rank = LottoRank.valueOf(matchCount, matchBonus);
+            lottoResults.addResult(rank);
+        }
+
+        return lottoResults;
+    }
```

Comment:
[코드] 저도 지금 생각난건데, 
for 구문 내부의 로직을 lotto 객체가 전부 다 스스로 처리하게 하는것은 어떻게 생각하시나요?


```suggestion
    public LottoResults aggregateLottoResults(WinningNumbers winningNumbers) {
        LottoResults lottoResults = new LottoResults();

        for (Lotto lotto : this.tickets) {
            LottoRank rank = lotto.evaluate(winningNumbers)
            lottoResults.addResult(rank);
        }

        return lottoResults;
    }

...

public class Lotto {

    public LottoRank evaluate(WinningNumbers winningNumbers) {
        int matchCount = countMatch(winningNumbers);
        boolean matchBonus = containsBonusNumber(winningNumbers);

        return LottoRank.valueOf(matchCount, matchBonus);
    }
}
```

Reply:
오 이렇게 정리해도 좋을 것 같습니다. 감사합니다!!